src/cli/near.rs

```diff
@@ -122,6 +182,89 @@ pub enum WriteCommand {
     FactoryUpdate {
         wasm_bytes_path: String,
     },
+    // deploy_code
+    DeployCode {
+        code_byte_hex: String,
+    },
+    // call
+    Call {
+        call_byte_hex: String,
+    },
```

Comment:
This seems redundant with `EngineCall` (and significantly harder to use than that one because you need to borsh-encode by hand)

Reply:
I will remove this.